address_book_menu.c

```diff
@@ -148,10 +148,272 @@ Status search_contact(AddressBook *address_book)
 
 Status edit_contact(AddressBook *address_book)
 {
-    /* Add the functionality for edit contacts here */
+    char save[200];
+	int choice;
+	int i;
+
+    AddressBook *address_book;
```

Comment:
Getting this error here. It's now line 176 in main. Not certain if there is a need to redeclare this here since it is passed into the function. 

address_book_menu.c:176:18: error: 'address_book' redeclared as different kind of symbol
  176 |     AddressBook *address_book;